src/js/services/configService.js

```diff
@@ -41,6 +41,22 @@ angular.module('copayApp.services').factory('configService', function(storageSer
     rates: {
       url: 'https://insight.bitpay.com:443/api/rates',
     },
+
+    pushNotifications: {
+      enabled: true,
+      url: 'http://192.168.1.111:8080',
```

Comment:
This can't be OK. Copay (client-side) cannot connect directly to the push server.